feature-libs/organization/administration/styles/_card.scss

```diff
@@ -9,10 +9,6 @@
     @extend %cardHeader !optional;
     @extend %cardMain !optional;
     @extend %cardSection !optional;
-
-    cx-org-toggle-status {
```

Comment:
anything in common with the delete functionality?

Reply:
https://github.com/SAP/spartacus/pull/10075#discussion_r534230659